patroni/ha.py

```diff
@@ -122,6 +122,14 @@ def bootstrap(self):
 
     def recover(self):
         self.recovering = True
+        if self.has_lock():
+            if self.patroni.config['master_start_timeout'] == 0:
```

Comment:
Should we try to update leader lock in other case (before making attempt to start in readonly)?